client/src/context/store.js

```diff
@@ -7,11 +9,15 @@ import {
   LOGIN_SUCCESS,
 } from './types';
 
+const ENDPOINT = '/';
+const socket = socketIOClient(ENDPOINT);
```

Comment:
Should this go into a useEffect? Has this been working fine for you?

Reply:
Yeah this has been working perfectly fine without being inside a useEffect